C7GameDataTests/SaveTest.cs

```
@@ -106,6 +106,11 @@ public async void LoadSampleSaves() {
 
 	[Fact]
 	public void LoadAllConquests() {
+		// When running the tests via github actions, civ3 isn't installed so we can't
+		// check the conquests directories.
+		string is_on_github = System.Environment.GetEnvironmentVariable("RUNNING_ON_GITHUB");
```

Instead of introducing the concept of being in ci (`RUNNING_ON_GITHUB`) to the codebase, perhaps LoadAllConquests should check if there's a folder at `conquests` and return if nothing is there?

---

I thought about that, but do we want the tests to silently pass if a dev hasn't set things up correctly?

---

Good point. In that case, let's check if CI is true or GITHUB_ACTIONS is in ENV instead of adding a new env variable? https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables

---

Nice idea, done.